package-lock.json

```diff
@@ -5,7 +5,7 @@
   "requires": true,
   "packages": {
     "": {
-      "version": "7.2.3",
+      "version": "7.2.4",
```

Comment:
I rarely use `npm`. I don't know why it would change unreleated things in the lockfile when running `npm install @testing-library/react@latest`

Reply:
It's cleaning up a bunch of sub-dependencies that are no longer unique to a package (e.g., `foo` requiring `bar@1.2.x` and `baz` requiring `bar@1.1.x` but then being upgraded to support 1.2.x).